test/e2e/flowaggregator_test.go

```diff
@@ -336,6 +341,11 @@ func checkBandwidthFromRecord(t *testing.T, record, bandwidth string) {
 	}
 }
 
+// TODO: Add a test that checks the functionality of Pod-To-External flow.
```

Comment:
this TODO seems out-of-place on this function, maybe it should have been part of the commit message